packages/lutils/lutils.1.51.2/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "erwan.jahier@imag.fr"
+authors: "Erwan Jahier"
+license: "CeCILL-2.1"
+homepage:
+  "https://gricad-gitlab.univ-grenoble-alpes.fr/verimag/synchrone/lutils/"
+bug-reports:
+  "https://gricad-gitlab.univ-grenoble-alpes.fr/verimag/synchrone/lutils/issues"
+doc: "http://www-verimag.imag.fr/DIST-TOOLS/SYNCHRONE/lustre-v6"
+
+depends: [
+  "ocaml" {>= "4.02"}
+  "dune"  {>= "2.0"}
+  "base-unix"
+  "ocamlfind"
+  "num"
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ]
+install: [make "install"]
```

Comment:
```suggestion
```
Your makefile script calls dune install underneath. This is unnecessary with opam as the build step already creates an .install file that is going to be used by opam.